### PR TITLE
Added target property for spa and docs button in opc-feedback

### PR DESCRIPTION
**Pull request by yamalameyooooo**
# Added target property of anchor tag for docs and spa buttons present in opc-feedback so that the docs and feedback page opens in a new tab

### 

# Explain the feature/fix

This PR allows the user to open spa and docs link in a different tab for opc-feedback

## Does this PR introduce a breaking change

No

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [ ] Does your component follow One Platform style guidelines?
- [ ] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [ ] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [x] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
- [x] Google Chrome [Latest 2 versions]
- [x] Mozilla Firefox [Latest 2 versions]
- [x] Microsoft Edge [Latest 2 versions]
- [x] Apple Safari [Latest 2 versions]
- [x] Microsoft Internet Explorer 11

### CODE REVIEW

**Review by riginoommen — CHANGES_REQUESTED** [review]
Update package-lock.json also

**Comment by yamalameyooooo** [comment]
@riginoommen updated package-lock too

File: packages/opc-feedback/src/opc-feedback.ts [hunk 1]
```diff
@@ -502,6 +502,7 @@ export class OpcFeedback extends LitElement {
             ${this.docs &&
             html`<li>
               <a
+                target="_blank"
```
Comment (hybridx) [inline packages/opc-feedback/src/opc-feedback.ts:505]: @sshamsudRH This should not be a default action, add a check and pass it down from the component attribute. There might be a use-case where we what this to open in the same page.

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/opc-feedback/src/opc-feedback.ts [hunk 2]
```diff
@@ -523,6 +524,7 @@ export class OpcFeedback extends LitElement {
             ${this.spa &&
             html`<li>
               <a
+                target="_blank"
```
Comment (hybridx) [inline packages/opc-feedback/src/opc-feedback.ts:527]: Same here

**Review by hybridx — CHANGES_REQUESTED** [review]

**Comment by yamalameyooooo** [comment]
@hybridx made the requested changes

**Comment by hybridx** [comment]
This is great work @sshamsudRH, can you please update the doc as well?

**Comment by yamalameyooooo** [comment]
@hybridx updated the documentation as well

**Review by hybridx — APPROVED** [review]

**Comment by hybridx** [comment]
LGTM approved 🎉

**Pull request merged**